packages/coclobas/coclobas.0.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Seb Mondet <seb@mondet.org>"
+authors: "Seb Mondet <seb@mondet.org>"
+homepage: "https://github.com/hammerlab/coclobas"
+bug-reports: "https://github.com/hammerlab/coclobas/issues"
+license: "Apache 2.0"
+dev-repo: "https://github.com/hammerlab/coclobas.git"
+build: [
+  [make "byte"]
+  [make "native"]
+  [make "META"]
+  [make "coclobas.install"]
+]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "solvuu-build" {build & >= "0.3.0"}
+  "nonstd"
+  "sosa"
+  "pvem_lwt_unix"
+  "cohttp"
+  "lwt"
```

Comment:
could you put a lower bound on a more recent version of lwt and cohttp? the solver is going old-skool for some reason...